projects/whydense/test_score_table.py

```diff
@@ -54,21 +54,30 @@
     "--format",
     "tablefmt",
     help="Table format",
-    type=click.Choice(choices=["grid", "latex"]),
+    type=click.Choice(choices=["grid", "latex_raw"]),
     show_default=True,
     default="grid",
 )
 def main(config, experiments, tablefmt):
-    # Use configuration file location as the project location.
-    project_dir = os.path.dirname(config.name)
-    project_dir = os.path.abspath(project_dir)
-    print("project_dir =", project_dir)
-    test_scores_table = [["Network", "Test Score", "Noise Score", "Noise Accuracy",
-                          "Nonzero Parameters", "Num Trials"]]
+
+    # The table we use in the paper
+    test_scores_table = [["Network", "Test Score", "Noise Score",
+                          "Params"]]
+
+    # A more detailed table
+    test_scores_table_long = [["Network", "Test Score", "Noise Score", "Noise Accuracy",
+                               "Total Entropy", "Nonzero Parameters", "Num Trials",
+                               "Session"]]
 
     # Load and parse experiment configurations
     configs = parse_config(config, experiments, globals_param=globals())
 
+    # Use the appropriate plus/minus sign for latex
+    if tablefmt == "grid":
+        pm = "±"
+    else:
+        pm = "$\\pm$"
```

Comment:
This value depends on `tablefmt`.

Reply:
True, but there are only two options (enforced by click).